JuliaSyntax/test/parser.jl

```diff
@@ -603,6 +603,7 @@ tests = [
         "macro \$f()    end"   =>  "(macro (call (\$ f)) (block))"
         "macro (\$f)()  end"   =>  "(macro (call (parens (\$ f))) (block))"
         "function (x) body end"=>  "(function (tuple-p x) (block body))"
+        "function (x)\n    body\nend"=>  "(function (tuple-p x) (block body))"
```

Comment:
Maybe also insert a involving explicitly a tuple `()` as the body?

E.g. near line 622-628, where in https://github.com/JuliaLang/JuliaSyntax.jl/pull/580 this test was inserted:
```
        "function (\n        f\n        )() end" =>  "(function (call (parens f)) (block))"
```
well now we want to test something like this (not sure if I got it quite right)
```
        "function (\n        x\n        )\n() end" =>  "(function (tuple-p x) (block (tuple-p)))"
```

Because for me, on master, this parses:
```julia
function(y)
  body
end
```
but this does not:
```julia
function(y)
  ()
end
```